scripts/new_project.kts

```diff
@@ -4,7 +4,15 @@ object NewProject {
 
     private const val ARGUMENT_DELIMITER = "="
     private const val DOT_SEPARATOR = "."
+    private val HELP_MESSAGE = """
```

Comment:
I think the name should be `helpMessage` and we should move it to near the other vals